src/google/protobuf/arena_impl.h

```diff
@@ -581,8 +581,8 @@ class PROTOBUF_EXPORT ThreadSafeArena {
 #pragma warning(disable : 4324)
 #endif
   using LifecycleId = uint64_t;
-  ABSL_CONST_INIT alignas(
-      kCacheAlignment) static std::atomic<LifecycleId> lifecycle_id_;
+  alignas(kCacheAlignment) ABSL_CONST_INIT static std::atomic<LifecycleId>
```

Comment:
We enforce clang-format formatting, so please break the line differently: with ABSL_CONST_INIT as the last thing before the break.  That is:

>     alignas(kCacheAlignment) ABSL_CONST_INIT
>         static std::atomic<LifecycleId> lifecycle_id_;

Reply:
Note that you can check formatting using the contextual popup within godbolt <https://twitter.com/jorgbrown/status/1602744303092002817> (screenshot taken from <https://godbolt.org/z/dxnMbY5x8> )

Reply:
done